pkg/didcomm/protocol/introduce/service.go

```diff
@@ -26,28 +31,180 @@ const (
 	AckMsgType = IntroduceSpec + "ack"
 )
 
+var (
+	_ service.Handler = &Service{}
```

Comment:
This line could use some explanation.

Reply:
I added it to be sure that Service satisfies service.Handler. If someone changed the interface service.Handler it will raise an error

Reply:
Why wouldn't that be in a test file?

Reply:
it is not a part of the test, it is a guarantee that service satisfies the common interface.

Reply:
We shouldn't instantiate a variable for this purpose.

Reply:
Done!